Cubical/Categories/Displayed/AdjointToReindex.agda

```diff
@@ -0,0 +1,173 @@
+{-# OPTIONS --safe #-}
+--
+module Cubical.Categories.Displayed.AdjointToReindex where
```

Comment:
Adjoint isn't specific enough. It's the left adjoint. (I'm not sure if the right adjoint always exists or if it requires a Conduche condition.)

This could also be called dependent sum, though I think that's confusing because to me a dependent sum is the special case where F is a projection.

Reply:
Yes, having a right adjoint is exactly being Conduché.